index.html

```diff
@@ -64,5 +64,159 @@ <h1 class="headline-alexandra_small">
+        <div class="right">
+          <h1>
+            Multi-Post<br />
+            Stories
+          </h1>
+          <div class="back">
+            <div>CANOPY</div>
+            <div class="circle"></div>
+            <div>Back End Dev</div>
+            <div class="circle"></div>
+            <div>2015</div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div><a href="">See Project</a></div>
+        </div>
+      </div>
+
+      <div class="tonic">
+        <div><img src="images/flex.png" alt="" /></div>
+        <div class="right">
+          <h1>Tonic</h1>
+          <div class="back">
+            <div>CANOPY</div>
+            <div class="circle"></div>
+            <div>Back End Dev</div>
+            <div class="circle"></div>
+            <div>2015</div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div><a href="">See Project</a></div>
+        </div>
+      </div>
+
+      <div class="tonic">
+        <div><img src="images/proffesional.png" alt="" /></div>
+        <div class="right">
+          <h1>
+            Multi-Post<br />
+            Stories
+          </h1>
+          <div class="back">
+            <div>CANOPY</div>
+            <div class="circle"></div>
+            <div>Back End Dev</div>
+            <div class="circle"></div>
+            <div>2015</div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div><a href="">See Project</a></div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
+      <h1>About Myself</h1>
+      <p class="description">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website. Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don’t hesitate to
+        contact me.
+      </p>
+
+      <p class="connect">lets connect</p>
+      <div class="icons">
+        <a href=""><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+        <a href=""><i class="fa fa-twitter" aria-hidden="true"></i></a>
+        <a href=""><img src="images/Vector.png" alt="medium-icon" /></a>
+        <a href=""><i class="fa fa-github" aria-hidden="true"></i></a>
+        <a href=""><i class="fa fa-hand-peace-o" aria-hidden="true"></i></a>
+      </div>
```

Comment:
- [ ] I think it would be better also for these list of icons to use a ```ul``` tag to group them, and ```li``` per icon; to explicitly mean a list of icons and also for the best HTML semantics 😎😎